aws/components/datafeed/setup.ftl

```diff
@@ -238,14 +238,23 @@
                                                 cmkKeyId,
                                                 streamBackupLoggingConfiguration )]
 
+        [#-- Establish bucket prefixes --]
+        [#local streamS3DestinationPrefix = solution.Bucket.Prefix]
+        [#local streamS3DestinationErrorPrefix = solution.Bucket.ErrorPrefix]
 
         [#switch (destinationLink.Core.Type)!"notfound" ]
+            [#case BASELINE_DATA_COMPONENT_TYPE]
+                [#if destinationLink.Core.SubComponent.Name == "opsdata"]
+                    [#local streamS3DestinationPrefix = formatPath(true, streamS3DestinationPrefix, core.FullAbsolutePath)]
+                    [#local streamS3DestinationErrorPrefix = formatPath(true, streamS3DestinationErrorPrefix, core.FullAbsolutePath)]
```

Comment:
```suggestion
                    [#local streamS3DestinationErrorPrefix = formatRelative(streamS3DestinationErrorPrefix, core.FullAbsolutePath)]
```

S3 doesn't support Absolute paths, only Relative paths
In S3 world / doesn't actually mean folder, they just use them to create folder like display in the console